src/odc/ODBAPISettings.cc

```diff
@@ -18,11 +18,13 @@
 #include "eckit/log/Log.h"
 #include "eckit/thread/ThreadSingleton.h"
 #include "eckit/utils/StringTools.h"
+#include "eckit/utils/Literals.h"
```

Comment:
This definitely requires a bump to the minimum version of eckit, in the CMakeLists.txt.

Reply:
Ah yes good catch!